cmd/minikube/cmd/start_flags.go

```diff
@@ -402,6 +403,18 @@ func getCNIConfig(cmd *cobra.Command) string {
 	return chosenCNI
 }
 
+func getExtraOptions() config.ExtraOptionSlice {
+	extraOptionVals := config.ExtraOptionSlice{}
+	val := viper.GetString(extraOptions)
+	if val == "" {
+		return extraOptionVals
+	}
+	if err := extraOptionVals.Set(val); err != nil {
```

Comment:
Will this be able to handle multiple flags/options?

Or, if an error occurs, will all the options be ignored?

Reply:
Update: please include the .Set() error message in the klog.Errorf().

Reply:
Added the error message!